src/main/java/git/GitCloner.java

```diff
@@ -21,10 +21,10 @@ public static void downloadRepo(String organization, String repoName, String bra
             // though the repo is cloned properly.
         }
 
-        try {
-            GitChecker.checkout(FileUtil.getRepoDirectory(organization, repoName), branchName);
-        } catch (RuntimeException e) {
-            System.out.println("Branch does not exist! Analyze terminated.");
+        try{
```

Comment:
The checkstyle fail because of the space before curly. 

You can auto fix them using `ALT + SHIFT + L` shortcut in intellij